apps/site-nft/src/components/Validate/index.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { defineMessage, FormattedMessage } from 'react-intl';
+import { Button, Form } from 'semantic-ui-react';
+import { AppContainer } from '../AppContainers';
+import { PageHeader } from '../PageHeader';
+import { log } from '@thecointech/logging';
+import icon from './images/icon_topup_big.svg';
+import { Upload } from '../Upload';
+import { getHash, readExif } from '../ProfileBuilder/SignAndUpload/sign';
+import { utils  } from 'ethers';
+import { getContract } from '@thecointech/nft-contract';
+
+const title = defineMessage({ defaultMessage: "Validate Image", description: "page title" });
+const description = defineMessage({ defaultMessage: "Validate the images cryptographic signature", description: "page instructions" });
+//const imageUri = defineMessage({ defaultMessage: "URI of image to validate", description: "Optional uri" });
```

Comment:
I took this back out because it's not worth hooking up.  We will eventually use it though, so only commented